pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -553,3 +559,17 @@ func SortedKeys(komposeObject kobject.KomposeObject) []string {
 	sort.Strings(sortedKeys)
 	return sortedKeys
 }
+
+//converts duration string to *int64 in seconds
+func DurationStrToSecondsInt(s string) *int64 {
+	if s == "" {
+		return nil
+	}
+	duration, err := time.ParseDuration(s)
+	if err != nil {
+		log.Warningf("Failed to parse duration:%v", s)
```

Comment:
nit: it would be great if this returns error, and at the place this function is called we emit `log.Warningf` since there we will also have info of which service has this problem.

The only problem with current approach is that we won't know which service is giving this particular warning!